### ISSUE TITLE
Version label redirects to non existing blog url

### ISSUE THREAD

**Issue by hasanhakkaev** [actualbudget/actual-server#17]
![image](https://user-images.githubusercontent.com/14126525/166122261-030842b3-5ecc-43d4-935b-4ec2be307631.png)

![image](https://user-images.githubusercontent.com/14126525/166122288-7db1f677-f1c4-41ce-b0c7-84788569f2b9.png)

**Comment by bzupnick** [comment]
@jlongster is directing to release notes still relevant due to the new state of Actual? Should this line be deleted entirely?

**Comment by jlongster** [comment]
Yeah will think about this. It probably should link directly to a github release, but we haven't set that part up yet.
